.gitignore

```diff
@@ -1,13 +1,209 @@
-cmake-build-debug/
```

Comment:
why were these removed?

Reply:
These files are no longer relevant to the project